console/web3ext/web3ext.go

```diff
@@ -933,6 +933,17 @@ web3._extend({
 			params: 1,
 			inputFormatter: [web3._extend.formatters.inputBlockNumberFormatter],
 		}),
+		new web3._extend.Method({
+			name: 'chainConfig',
+			call: 'klay_chainConfig',
+			params: 0,
```

Comment:
Since `governance.chainConfig` is a property, not a function, I think we should implement this as a property.

Reply:
Right Thanks!